src/lib/pseudo.js

```diff
@@ -1,27 +1,45 @@
 'use strict';
 
-// Recursively walk an AST node searching for content leaves
-export function walkContent(node, fn) {
-  if (typeof node === 'string') {
-    return fn(node);
+// Walk an entry node searching for content leaves
+export function walkEntry(entry, fn) {
```

Comment:
that's not enough. The problem is not about expression. The problem is that the shape of en entry is the same as the shape of the Hash. 

And I do recurse into value and attrs, but I use walkValue instead of walkEntry.